src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -58,58 +66,197 @@
           </v-btn>
         </v-col>
       </v-row>
-
-      <v-row
-        class="mt-4"
-        no-gutters
+      <v-form
+        v-if="jurisdiction && !isMrasJurisdiction"
+        ref="manualFormRef"
+        lazy-validation
+        @submit.prevent
       >
-        <v-col
-          cols="12"
-          sm="3"
-        >
-          <label>Identifying Number</label>
-        </v-col>
+        <!-- Identifying Number -->
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          />
 
-        <v-col
-          cols="12"
-          sm="7"
-        >
-          ** Identifying number input goes here **
-        </v-col>
+          <v-col
+            cols="12"
+            sm="7"
+          >
+            <v-text-field
+              id="business-name"
+              v-model="business.identifier"
+              filled
+              label="Identifying Number"
+              :rules="getShowErrors ? identifyingNumberRules : []"
+            />
```

Comment:
You might want to add `hide-details="auto"` to reduce the whitespace below the test field when no hint/error is shown. Then add explicit whitespace (eg, "mt-4") between components so it looks nice.

Reply:
done

Reply:
Looks good, though it looks like a bit more whitespace is needed according to the Figma:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/a5a8e7b6-444f-4dbf-8165-a4f62e9dcdb5)

Also, looks like you need to add a hint to the Business Number input.